#Set the behavior to "change to active desktop" when a window of a different workspace is selected.

Index: avant-window-navigator/data/awn.schema-ini.in
===================================================================
--- avant-window-navigator.orig/data/awn.schema-ini.in	2009-04-25 01:12:12.000000000 +0200
+++ avant-window-navigator/data/awn.schema-ini.in	2009-04-25 01:12:29.000000000 +0200
@@ -124,7 +124,7 @@
 description = The relative horizontal position of the bar.  0 is the left edge of the screen, 1 is the right edge.
 [window_manager/activate_behavior]
 type = int
-default = 0
+default = 1
 description = Behavior when window on a different workspace is activated. (0 for default, 1 - change active workspace, 2 - move to current workspace)
 [window_manager/show_all_windows]
 type = bool
